features/admin.application-templates.v1/constants/templates.ts

```diff
@@ -30,10 +30,16 @@ export class ApplicationTemplateConstants {
             displayOrder: 0,
             id: "DEFAULT"
         },
+        {
+            description: "applicationTemplates:categories.technology.description",
+            displayName: "applicationTemplates:categories.technology.displayName",
```

Comment:
Let's remove this key, if the model has it as required field let's change it as optional

Reply:
resolved via https://github.com/wso2/identity-apps/pull/8383/commits/020f2ec3f444ecedd68abaac58095cba7fdd839f